teuthology/task/install/rpm.py

```diff
@@ -19,6 +19,7 @@ def _remove(ctx, config, remote, rpm):
     """
     remote_os = remote.os
     dist_release = remote_os.name
+    dist_version = float(remote_os.version)
```

Comment:
if `remote_os.version` is a string, it is better to use `remote_os.version.split(".")` and then compare:
`dist_version >= ["15"] and dist_version < ["42"]`
or even better to convert int array:
```
dist_version = [int(x) for x in remote_os.version.split(".")]
.
.
.
if dist_release in ['opensuse', 'sle'] and \
   dist_version >= [15] and dist_version < [42]:
.
.
.
```

Reply:
I don't know. Having a version number be an array seems really weird. But I'll convert it into an integer and call it "dist_major_version" if you like.